drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/editor/ScenarioSimulationEditorPresenter.java

```diff
@@ -291,7 +307,7 @@ protected void loadContent() {
     void populateRightPanel() {
         // Execute only when RightPanelPresenter is actually available
         getRightPanelPresenter().ifPresent(presenter -> {
-            presenter.onDisableEditorTab();
+            // presenter.onDisableEditorTab();
```

Comment:
if this is commented out, then does the dock get removed? If you open a DRL editor after this does the right side dock stay visible?